### ISSUE TITLE
Make ckanext-qa translatable

### ISSUE THREAD

**Issue by KrzysztofMadejski** [ckan/ckanext-qa#17]
Missing i18n in many templates in templates_new.

Commit will follow along with babel configuration:

``` bash
cd ckanext-qa
python setup.py extract_messages
python setup.py update_catalog -l LANG
# or #python setup.py init_catalog -l LANG

# Translate i18n/LANG/LC_MESSAGES/ckanext-qa.po
```

**Comment by KrzysztofMadejski** [comment]
Just cherrypick this commit.

**Comment by KrzysztofMadejski** [comment]
That's done and merged in #50
